play-media-native/features/SportPicker/SportPicker.tsx

```diff
@@ -43,6 +43,16 @@ export const SportPicker = ({ onChange, sports, initialValue = '' }: SportPicker
     [onChange]
   );
 
+  const isSelected = useCallback(
```

Comment:
Is this needed? 

Why not just 

`
const [value, setValue] = useState(initialValue);
`
.....
`<RadioButton.Group onValueChange={handleChange} value={value}>
`
.....
`
backgroundColor:
   value === sport.title ? theme.colors.yellow.DEFAULT : theme.colors.white.DEFAULT
`

that is easier to read?